core/src/window_service.rs

```diff
@@ -137,17 +141,50 @@ impl WindowServiceMetrics {
     }
 }
 
+fn should_send_index_and_erasure_conflicts(shred_slot: Slot, root_bank: &Arc<Bank>) -> bool {
+    match root_bank
+        .feature_set
+        .activated_slot(&feature_set::index_erasure_conflict_duplicate_proofs::id())
+    {
+        None => false,
+        Some(feature_slot) => {
+            let epoch_schedule = root_bank.epoch_schedule();
+            let feature_epoch = epoch_schedule.get_epoch(feature_slot);
+            let shred_epoch = epoch_schedule.get_epoch(shred_slot);
+            // Has a 1 epoch delay, as we don't have enough information
+            // on the epoch boundary of the feature activation
+            feature_epoch < shred_epoch
+        }
+    }
+}
```

Comment:
You can make this function:
https://github.com/solana-labs/solana/blob/2971e84ec/turbine/src/cluster_nodes.rs#L514-L526
and reuse it here.